include/rocksdb/env.h

```diff
@@ -1352,7 +1353,8 @@ extern void Fatal(Logger* info_log, const char* format, ...)
 // A utility routine: write "data" to the named file.
 extern Status WriteStringToFile(Env* env, const Slice& data,
                                 const std::string& fname,
-                                bool should_sync = false);
+                                bool should_sync = false,
+                                const IOOptions* io_options = nullptr);
```

Comment:
Pointer is used here cuz IOOptions from file_system.h can't be included in Env so we can't default initialize it to avoid breaking change to public API.